Example/Snabble/AccountViewController.swift

```diff
@@ -5,11 +5,26 @@
 //  Copyright © 2021 snabble. All rights reserved.
 //
 
-import UIKit
+import Foundation
+import SnabbleSDK
+import SwiftUI
 
-final class AccountViewController: UIViewController {
-    init() {
-        super.init(nibName: nil, bundle: nil)
+public struct ProfileView: View {
+    @ObservedObject public var viewModel: DynamicViewModel
+
+    public var body: some View {
+        NavigationView {
+            WidgetContainer(viewModel: viewModel, widgets: viewModel.widgets)
+                .navigationBarTitle(Asset.localizedString(forKey: "Sample.Profile.title"), displayMode: .inline)
+        }
+        .navigationViewStyle(.stack)
+   }
+}
```

Comment:
Ist es gewollt das der Teil nicht verwendet wird?

Reply:
Das muss noch in der SampleApp einbaut werden. SwiftUI Beispiel...

Reply:
Magst Du das machen?

Reply:
Kann ich gerne nach dem Mittag ausprobieren. Sperrt für mich den PR nicht.